hack/bundle-automation/copy-config.yaml

```diff
@@ -4,6 +4,6 @@
     - name: "cluster-permission"
       chart-path: "chart/"
       always-or-toggle: "toggle"
-      branch: "release-2.10"
+      branch: "main"
```

Comment:
Using `main` branch as reference until `release-2.11` branch is created for `cluster-permission`.

Reply:
Verified that the `release-2.11` was created. Updated the config file to reflect those changes.